### ISSUE TITLE
cb() never called error with npm install on ionic project directory

### ISSUE THREAD

**Issue by RyeGuy12** [npm/npm#17345]
I'm opening this issue because npm is producing an incorrect install.

When I run the command npm install in an ionic project directory I got the error

error cb() never called! 

#### How can the CLI team reproduce the problem?

You can reproduce this issue if you have a pluralsight account, go to the course:

Building Mobile Apps with Ionic 2, Angular 2, and TypeScript
by Steve Michelotti

and download the exercise files. Navigate to the exercise files chapter 8 project directory (elite-schedule-app) in the terminal. Upon arrival enter command **ionic serve** it will then ask if you wish to install the project directories and select yes. This is when the error occurs.

ERROR LOG:

1 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'install' ]
2 info using npm@5.0.3
3 info using node@v8.1.2
4 verbose npm-session 1dada19cd936fd55
5 silly install runPreinstallTopLevelLifecycles
6 silly preinstall elite-schedule-app@
7 info lifecycle elite-schedule-app@~preinstall: elite-schedule-app@
8 silly lifecycle elite-schedule-app@~preinstall: no script for preinstall, continuing
9 silly install loadCurrentTree
10 silly install readLocalPackageData
11 silly install loadIdealTree
12 silly install cloneCurrentTreeToIdealTree
13 silly install loadShrinkwrap
14 silly install loadAllDepsIntoIdealTree
15 error cb() never called!

**Comment by brianantonelli** [comment]
I'm experiencing the same issue. First I was alerted my shrinkwrap was out of date so I removed it and re-ran `npm shrinkwrap`

Now when I run a `npm install` I get:

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'install' ]
2 info using npm@5.0.0
3 info using node@v4.4.7
4 silly install runPreinstallTopLevelLifecycles
5 silly preinstall alks@2.10.2
6 info lifecycle alks@2.10.2~preinstall: alks@2.10.2
7 silly lifecycle alks@2.10.2~preinstall: no script for preinstall, continuing
8 silly install loadCurrentTree
9 silly install readLocalPackageData
10 silly install loadIdealTree
11 silly install cloneCurrentTreeToIdealTree
12 silly install loadShrinkwrap
13 silly install loadAllDepsIntoIdealTree
14 error cb() never called!
15 error This is an error with npm itself. Please report this error at:
16 error <https://github.com/npm/npm/issues>
```

Same issue when I try to update NPM:
```
🍔  npm i -g npm
npm ERR! cb() never called!
```

**Comment by RyeGuy12** [comment]
Yep I have not found anything yet that will help me get through this

**Comment by brianantonelli** [comment]
Installing Node again on OSX (https://nodejs.org/en/download/) resolved the issue for me.

**Comment by AurelienC** [comment]
Same error #17222
